mappings/net/minecraft/entity/ai/brain/Behaviour.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/unmapped/C_rcqaryar net/minecraft/entity/ai/brain/Behaviour
```

Comment:
oh dear this is quite british

Reply:
didn't even notice that was the entire class wow